### PR TITLE
fix: 兼容性

### PR DESCRIPTION
**Pull request by HeeMingYang**
Description: ocr功能动态判断是否有效

Log: 系统未安装ocr包时，移除ocr按钮

### CODE REVIEW

**Review by hundundadi — COMMENTED** [review]
[是否满足兼容性要求] Y
[是否满足commit提交规范] Y
[是否满足编码规范] Y
[Review结论] Pass
[Fail原因] N/A

**Review by pengfeixx — APPROVED** [review]
/+1

**Comment by wineee** [comment]
尽量避免硬编码，可以参考 https://github.com/linuxdeepin/deepin-screen-recorder/pull/207

File: src/widgets/subtoolwidget.cpp [hunk 1]
```diff
@@ -712,7 +712,9 @@ void SubToolWidget::initShotLabel()
     m_ocrButton->setFixedSize(MIN_TOOL_BUTTON_SIZE);
     installTipHint(m_ocrButton, tr("Extract Text"));
 #ifdef  OCR_SCROLL_FLAGE_ON
-    btnList.append(m_ocrButton);
+     if (QFile("/usr/bin/deepin-ocr").exists()) {
```
Comment (wineee) [inline src/widgets/subtoolwidget.cpp:715]: 尽量避免硬编码，可以参考 https://github.com/linuxdeepin/deepin-screen-recorder/pull/207

**Pull request merged**